docs/template_data.state

```diff
@@ -1,2 +1,2 @@
 This file stores hash of website templates to trigger Netlify rebuild when something changes, e.g. new linter is added.
-4896b58a5fb540cfca8bbcec4ef437b426bac0cbb1a4bd8dda3fe3a2a60f0c29
\ No newline at end of file
+5e1b1118687c421b50605b619e5497b4058abe0991eb345f9b450f3e1de76163
```

Comment:
I think netlify will check this hash and re-build the website, but not 100% sure it will be triggered automatically.

Reply:
Netlify triggers rebuild only if anything has changes in `docs/` folder: https://golangci-lint.run/contributing/website/#trigger-website-rebuild

Reply:
yeah, I saw this condition as well. But seems like latest version is not available in public website